### ISSUE TITLE
Display commit message when :Gblame is first called

### ISSUE THREAD

**Issue by tommcdo** [tommcdo/vim-fugitive-blame-ext#1]
Message is currently only displayed when the cursor is moved, so it does not show the message for the commit under the cursor when the blame window is initially opened. I've tried adding the BufEnter event, but it causes a noticeable flicker -- it seems that the git command runs before fugitive gets a chance to set nowrap and scrollbind.

**Comment by kassio** [comment]
I tried with the `BufReadPost`, and it seems to work fine. 😊

**Comment by kassio** [comment]
No, actually it shows a wrong message, sorry.. :(

**Comment by tommcdo** [comment]
Yeah, this one is tricky because I think other `:autocmd`s from fugitive.vim are involved, so getting the timing right is difficult.

Maybe the solution is to get this built into fugitive.vim itself :P

**Comment by kassio** [comment]
I think it's a good suggestion, wdyt to open a PR on the original? 😄

**Comment by tommcdo** [comment]
Actually, this plugin is sort of a proof-of-concept for [a feature request I opened a long time ago](https://github.com/tpope/vim-fugitive/issues/405). @tpope has a similar feature implemented on a branch, so maybe he needs some encouragement to merge it in :smile:
